stroom-dashboard/stroom-dashboard-client/src/main/java/stroom/dashboard/client/text/TextPresenter.java

```diff
@@ -260,9 +260,6 @@ private List<TextRange> getHighlights(final String input, final Set<String> high
                                         new DefaultLocation(lineNo, colNo),
                                         new DefaultLocation(lineNo, colNo + highlightLength - 1)); // inclusive
                                 highlights.add(hl);
-
-                                i += highlightLength;
```

Comment:
I think rather than removing this you may need to add
```
colNo += highlightLenght;
```
I think `i` is being advanced by the length of the highlight to save it going over each char in the highlight when it doesn't need to.  As `i` is going up by highlightLength and colNo is just going up by one they are getting out of sync.